client/src/action_creators.ts

```diff
@@ -201,7 +169,7 @@ export function fetchBits() {
 export function fetchNextPage() {
   return function (dispatch: Dispatch<any>, getState: () => Map<string, any>) {
     const offset = (getOffset(history.location.search) || 0)
-        + (getPageSize(history.location.search) || DEFAULT_PAGE_SIZE);
+      + (getPageSize(history.location.search) || DEFAULT_PAGE_SIZE);
```

Comment:
And... 2 spaces for continued statements??

Reply:
Yeah that's pretty standard for js/ts.